static/src/javascripts/projects/common/utils/fastdom-errors.js

```diff
@@ -0,0 +1,19 @@
+// Record errors within fastdom:
+// https://github.com/wilsonpage/fastdom#exceptions
+
+define([
+    'fastdom',
+    'raven'
+],
+function (
+    fastdom,
+    raven
+) {
+    fastdom.onError = function (error) {
+        raven.captureException(new Error('Error inside fastdom: ' + error), {
```

Comment:
As discussed, no need to create a new error here, just throw the one passed to this callback.